site/_data/i18n/landing.yml

```diff
@@ -0,0 +1,5 @@
+landing_sections:
```

Comment:
Nit: as the filename already sets the scope, that we are in landing I think we can simply use `sections` as a key here.

Reply:
Yep, my bad, moved it from common.yml after review and didn't change the names